### PR TITLE
frontend/fix/#1120: added cross btn for try with tip popup

### PR DESCRIPTION
**Pull request by ashkriti**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
The tip bottom sheet(when user tries to cancel on-going search and search again with a tip)

## Motivation and Context
https://github.com/nammayatri/nammayatri/issues/1120

## How did you test it?
Tested Manually

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

## Screenshots

Screenshot 2023-06-14 at 2 21 09 PM

Screenshot_2023-06-06-20-32-10-68_88a774e4ba75b7addb8b842e78164a94

### CODE REVIEW

**Review by MercyQueen — COMMENTED** [review]
@ashkritidewan2000  can we add the images in main directory itself instead of adding it in each merchant directory separately?

**Review by MercyQueen — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Components/PopUpModal/View.purs [hunk 1]
```diff
@@ -66,8 +66,19 @@ view push state =
...
+    ][ linearLayout
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , margin (Margin 0 0 14 13)
+        , gravity RIGHT
+        , visibility state.imageVisibility
+        ][ imageView
+            [ height $ V 21
+            , width $ V 21
+            , imageWithFallback "ny_ic_dismiss,https://assets.juspay.in/nammayatri/images/user/ny_ic_dismiss.png"
```
[the remark this answers is not in the document]
Comment (ashkriti) [reply Frontend/ui-common/src/Components/PopUpModal/View.purs:78]: Done

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs [hunk 2]
```diff
@@ -355,6 +355,7 @@ logOutPopUpModelConfig state =
         popUpConfig'
     ST.TipsPopUp -> PopUpModal.config{
           optionButtonOrientation = "VERTICAL"
+          , imageVisibility = if(isLocalStageOn ST.QuoteList) then GONE else VISIBLE
```
[the remark this answers is not in the document]
Comment (ashkriti) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs]: Done

**Review by MercyQueen — COMMENTED** [review]
added comments

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**